lib/parser/typeParser.ts

```diff
@@ -72,6 +81,9 @@ export function parseEvmType(rawType: string): EvmType {
       return new BytesType(1);
     case "bytes":
       return new ArrayType(new BytesType(1));
+    case "tuple":
+      if (!components) throw new Error("Tuple specified without components!");
```

Comment:
just use invariant lib as we do in other places of the codebase. You will have to use components! in next line to let compiler know that we handled it

Reply:
@krzkaczor - sorry I grep'd for invariant and couldn't find any references to that library in the project.  Would you like me to include it?

Reply:
Sorry, I was mistaken that we already use it in this project. Okay don't worry about it ;) sorry about the confusion, your solution is fine anyway.